boards/weio/include/periph_conf.h

```diff
@@ -88,44 +90,30 @@ extern "C" {
  * @brief PWM configuration
  * @{
  */
-#define PWM_0_EN            1
-#define PWM_0_CHANNELS      3
-#define PWM_1_EN            1
-#define PWM_1_CHANNELS      3
-#define PWM_NUMOF           (2U)
-
-/* PWM0 common configuration */
-#define PWM_0_DEV           LPC_CT16B0
-#define PWM_0_CLK           BIT7
-/* PWM_0 channel configuration */
-#define PWM_0_CH0_EN        1
-#define PWM_0_CH0_IOCON     LPC_IOCON->PIO1_13
-#define PWM_0_CH0_AF        0x82
-
-#define PWM_0_CH1_EN        1
-#define PWM_0_CH1_IOCON     LPC_IOCON->PIO1_14
-#define PWM_0_CH1_AF        0x82
-
-#define PWM_0_CH2_EN        1
-#define PWM_0_CH2_IOCON     LPC_IOCON->PIO1_15
-#define PWM_0_CH2_AF        0x82
-
-/* PWM1 common configuration */
-#define PWM_1_DEV           LPC_CT32B0
-#define PWM_1_CLK           BIT9
-/* PWM_1 channel configuration */
-
-#define PWM_1_CH0_EN        1
-#define PWM_1_CH0_IOCON     LPC_IOCON->PIO1_24
-#define PWM_1_CH0_AF        0x81
-
-#define PWM_1_CH1_EN        1
-#define PWM_1_CH1_IOCON     LPC_IOCON->PIO1_25
-#define PWM_1_CH1_AF        0x81
-
-#define PWM_1_CH2_EN        1
-#define PWM_1_CH2_IOCON     LPC_IOCON->PIO1_26
-#define PWM_1_CH2_AF        0x81
+static const pwm_conf_t pwm_config[] = {
+    {
+        .dev     = LPC_CT16B0,
+        .pins    = {
+            &LPC_IOCON->PIO1_13,
+            &LPC_IOCON->PIO1_14,
+            &LPC_IOCON->PIO1_15
+        },
+        .clk_bit = BIT7,
+        .af      = 0x02
```

Comment:
The question is the other way around: why was it 0x81 before -> looking at the ref manual it states, that bit7 (the '8') is unused...